test/unit/spec/RedirectUtil_spec.js

```diff
@@ -0,0 +1,101 @@
+define([
+  'util/RedirectUtil'
+],
+function (RedirectUtil) {
+
+  describe('util/RedirectUtil', function () {
+
+    describe('buildDynamicForm', function() {
+      it('returns an element', function() {
+        var form = RedirectUtil.buildDynamicForm('https://example.com');
+        expect(form).not.toBeUndefined();
+      });
+
+      it('returns a form whose method is GET', function() {
+        var form = RedirectUtil.buildDynamicForm('https://example.com/path?foo=bar');
+        expect(form.method).toBe('get');
+      });
+
+      it('returns a form whose action is the URL minus query', function() {
+        var form = RedirectUtil.buildDynamicForm('https://example.com/path?foo=bar');
+        expect(form.action).toBe('https://example.com/path');
+      });
+    });
+
+    describe('buildDynamicForm redirection', function() {
+      function createTestIframe() {
+        var frame = document.createElement('iframe');
+        frame.style = 'display: none;';
+        document.body.appendChild(frame);
+
+        frame.contentDocument.open();
+        frame.contentDocument.write("<html><body></body></html>");
+        frame.contentDocument.close();
+        
+        return frame;
+      }
+
+      var wait = 25;
```

Comment:
why needs wait?

Reply:
The navigation is not synchronous, it takes a few cycles for the iframe URL to change after `form.submit()`. In my testing 10ms was too low, 25ms seemed to be okay.

Reply:
Probably need to dig into more (maybe to mock the timer).
Fighting with time is always not a good practice at first place.